pom.xml

```diff
@@ -115,8 +115,8 @@
                     // Therefore, the project uses jdk1.7 uniformly,
                     // but the server module is specified separately for version 1.8
                     -->
-                    <source>1.7</source>
-                    <target>1.7</target>
+                    <source>1.8</source>
```

Comment:
Ignored the precautions requirements, which will force the client environment to be a 1.8 JDK